Makefile

```diff
@@ -0,0 +1,45 @@
+checkstyle:
```

Comment:
I haven't prefixed `notificationbuilder-` to all the make command. Thinking we can wait until the next UIComponent is added to edit these command and the ci files.
Let me know